mos_tests/rabbitmq_oslo/test_restarts.py

```diff
@@ -236,6 +235,32 @@ def consume_msg(remote, cfg_file_path):
     num_of_msg_consumed = int(re.findall('\d+', out_consume)[0])
     return num_of_msg_consumed
 
+
+def rabbit_rpc_server_start(remote, cfg_file_path):
+    logger.debug('Start [oslo_msg_check_server] on %s.' % remote.host)
+    cmd = 'oslo_msg_check_server --nodebug --config-file {0}'.format(
+        cfg_file_path)
+    remote.execute_async(cmd)
```

Comment:
`<&- >/dev/null 2>&1 &` - add it to end of command and it will run in background even if you close SSH connection.